front/app/components/UI/Loading/index.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { Suspense } from 'react';
+import FullPageSpinner from '../FullPageSpinner';
+
+interface Props {
+  // If we're in the admin, the loader will take into account
+  // the admin navbar's width.
+  admin?: boolean;
```

Comment:
Instead of doing this with a prop and passing it everywhere (including having to think about it every time a new route is created) I believe it would be easier to just check directly in the component whether it's an admin route or not.